ledger/acctonline.go

```diff
@@ -467,6 +470,19 @@ func (ao *onlineAccounts) postCommit(ctx context.Context, dcc *deferredCommitCon
 		ao.baseOnlineAccounts.write(persistedAcct)
 	}
 
+	for i := uint64(0); i < offset; i++ {
+		roundDelta := ao.deltas[i]
+		for j := 0; j < roundDelta.Len(); j++ {
+			addr, acctDelta := roundDelta.GetByIdx(j)
+			persistedAcct := persistedOnlineAccountData{
+				addr:  addr,
+				round: ao.cachedDBRoundOnline + basics.Round(i+1),
+			}
+			persistedAcct.accountData.SetCoreAccountData(acctDelta)
+			ao.onlineAccountsCache.writeFront(persistedAcct)
```

Comment:
the main concern there is we write ALL modified accounts into online accounts cache.
It needs to be pruned to only 1) new online accounts 2) prev online accounts went offline 3) online accounts with changed state. Please use updatedPersistedOnlineAccounts as a source for pruning deltas since it has all required info except the updRound